packages/qt5_base.rb

```diff
@@ -6,7 +6,7 @@ class Qt5_base < Package
   version 'kde-5.15.14-9f9a56d-icu75.1'
   license 'GPL-2'
   compatibility 'x86_64 aarch64 armv7l'
-  min_glibc '2.37'
+  # min_glibc '2.37'
```

Comment:
No rebuild necessary to make less than 2.37 compatible?

Reply:
I'm in the middle of an updated rebuild for it right now... But this is a dependency for something  `pangomm_2_48` needs but doesn't link to.

Reply:
Should this PR be moved to draft then?